### PR TITLE
Increase ShortMaxWait for more stability

### PR DESCRIPTION
**Pull request by MarcelMue**
towards https://github.com/giantswarm/giantswarm/issues/4016

### CODE REVIEW

File: pkg/framework/backoff.go [hunk 1]
```diff
@@ -6,7 +6,7 @@ import (
 
 const (
 	LongMaxWait  = 40 * time.Minute
-	ShortMaxWait = 2 * time.Minute
+	ShortMaxWait = 4 * time.Minute
```
Comment (MarcelMue) [inline pkg/framework/backoff.go]: Making this longer for chart-operator. Seems to be working - uncertain why we need more time now though.
See https://circleci.com/gh/giantswarm/chart-operator/tree/update-e2e-harness

**Review by indyfree — APPROVED** [review]
LGTM. Hopefully the flapping reduces with that. Would be nice to find out why we suddenly need more time though.

**Comment by xh3b4sd** [comment]
Increasing timeouts and hoping for the problem to be magically fixed might not be what we want. Just saying.

**Comment by MarcelMue** [comment]
Sure, this is to increase our quality of life right now as I don't want to invest too much time into this. I will keep the issue open to figure out what's actually wrong.

**Pull request closed without merging**